TeachingRecordSystem/src/TeachingRecordSystem.Core/Events/Models/TrnRequestMetadata.cs

```diff
@@ -63,7 +63,7 @@ public static TrnRequestMetadata FromModel(Core.DataStore.Postgres.Models.TrnReq
             Country = model.Country,
             TrnToken = model.TrnToken,
             ResolvedPersonId = model.ResolvedPersonId,
-            Matches = model.Matches is not null ?
+            Matches = model.Matches is not null && model.Matches.MatchedRecords is not null ?
```

Comment:
I don't know whether I just set up the json data wrongly in the DB but this was throwing without the etra check

Reply:
..ah - I probably didn't have a []